packages/core-components/src/components/ContactTable.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+export type ContactTableItemType = {
+  title: string;
+  description: string;
+  address: string;
+  phone: string;
+  email: string;
+};
+
+type ContactTableProps = {
+  contactTableItem: ContactTableItemType;
+};
+
+const ContactTable: React.FC<ContactTableProps> = ({ contactTableItem }) => {
+  const titleFirstLine = contactTableItem.title
```

Comment:
title could be undefined/null so it will be better if we have null/undefined check and return a default string( empty should be fine)

Reply:
Also, I find it not reliable when you split on 3rd word like this. We could posibly use English and this will break the title. We could split it into 2 props (subTitle and title) for this